core/controlplane/config/templates/stack-template.json

```diff
@@ -258,7 +279,8 @@
               "Effect": "Allow",
               "Principal": {
                 "Service": [
-                  "ec2.{{.Region.PublicDomainName}}"
+                  "ec2.{{.Region.PublicDomainName}}",
+                  "autoscaling.{{.Region.PublicDomainName}}"
```

Comment:
Without this, the stack creation fails because AWS tries to publish a test message to the topic.